fusor-ember-cli/app/controllers/review/summary.js

```diff
@@ -19,6 +19,11 @@ export default Ember.Controller.extend({
     return ('http://' + this.get('model.openstack_undercloud_ip_addr'));
   }.property('model.openstack_undercloud_ip_addr'),
 
+  overcloudUrl: function() {
+    return ('http://' + this.get('model.openstack_overcloud_password') + '/dashboard/admin');
```

Comment:
Currently it returns the entire URL, but I can pare it down to just the IP address. I think that will be better; if I can ever figure out how to authenticate to the heat api and get it that way instead it will return just the IP address, so probably better to go that way now and save a change in the future.